ember-cli-build.js

```diff
@@ -9,14 +9,14 @@ var broccoliAutoprefixer = require('broccoli-autoprefixer');
 var autoprefixerOptions = require('./build/autoprefixer-options');
 
 module.exports = function(defaults) {
-  var demoAppCssTree = new BroccoliSass(['src/demo-app'], './demo-app.scss', 'demo-app.css');
+  var demoAppCssTree = new BroccoliSass(['src/demo-app'], './demo-app.scss', 'demo-app/demo-app.css');
   var componentCssTree = getComponentsCssTree();
   var angularAppTree = new Angular2App(defaults);
 
   return mergeTrees([
     angularAppTree.toTree(),
     componentCssTree,
-    demoAppCssTree,
+    demoAppCssTree
```

Comment:
The trailing comma is there on purpose. Keeping them around does two things:
1) Less noise on diffs when you add / remove lines
2) `clang-format` formats object/array literals differently with/without the trailing comma.